develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/00-using-liferay-util-taglibs-intro.markdown

```diff
@@ -0,0 +1,22 @@
+# Using the Liferay Util Taglib [](id=using-the-liferay-util-taglib)
+
+The Liferay Util taglib is used to pull other resources into a portlet or theme. 
+You can use it to specify which resources to insert at the bottom or top of the 
+page's HTML. 
+
+To use the Liferay-Util taglib, add the following declaration to your JSP:
+
+    <%@ taglib prefix="liferay-util" uri="http://liferay.com/tld/util" %>
+
+The Liferay-Util taglib is also available via a macro for your FreeMarker theme 
+templates and web content templates. Follow this syntax:
+
+    <@liferay_util["tag-name"] attribute="string value" attribute=10 />
+
+Each Liferay Util tag has an optional `outputKey` attribute. If several portlets 
```

Comment:
This is not true at all. Only `body-bottom`, `body-top`, `html-bottom` and `html-top` have that as it's specific to their behaviour

Reply:
hmmm... thanks for letting me know!